src/jit/emitxarch.cpp

```diff
@@ -10532,9 +10545,29 @@ size_t              emitter::emitOutputInstr(insGroup* ig, instrDesc* id, BYTE**
 
         // If the method returns a GC ref, mark EAX appropriately
         if (id->idGCref() == GCT_GCREF)
+        {
             gcrefRegs |= RBM_EAX;
-        else if  (id->idGCref() == GCT_BYREF)
+        }
+        else if (id->idGCref() == GCT_BYREF)
+        {
             byrefRegs |= RBM_EAX;
+        }
+        
+#ifdef FEATURE_MULTIREG_RET
+        // If is a multi-register return method is called, mark RDX appropriately (for System V AMD64).  
```

Comment:
Note that FEATURE_MULTIREG_RET is always defined.  What we need to check is whether it is 0 (disabled) or 1 (enabled)